app/helpers/batches_helper.rb

```diff
@@ -36,12 +91,6 @@ def output_aliquot(xml, aliquot) # rubocop:todo Metrics/AbcSize
     end
   end
 
-  def workflow_name(batch)
```

Comment:
Wasn't unused, but the logic would have no affect for existing pipelines, so removed.